app/src/main/java/com/github/h3lp3rs/h3lp/storage/Storages.kt

```diff
@@ -8,7 +8,7 @@ import java.lang.Boolean.parseBoolean
  * Enumeration of all useful (local) storages in H3LP
  */
 enum class Storages() {
-    USER_COOKIE(), MEDICAL_INFO(), SKILLS(),EMERGENCIES_RECEIVED();
+    USER_COOKIE(), MEDICAL_INFO(), SKILLS(),EMERGENCIES_RECEIVED(),FORUM_THEMES;
```

Comment:
Is that where we store the notification preferences ? The name "Theme" does not seem like the most appropriate